sdk/openai/openai/test/public/utils/utils.ts

```diff
@@ -227,11 +227,10 @@ export async function get(url: string, recorder: Recorder): Promise<PipelineResp
   return sendRequestWithRecorder(request, recorder);
 }
 
-export function createAzureCognitiveSearchExtension(): AzureCognitiveSearchChatExtensionConfiguration {
+export function createAzureSearchExtension(): AzureSearchChatExtensionConfiguration {
   return {
-    type: "AzureCognitiveSearch",
+    type: "azure_search",
     endpoint: assertEnvironmentVariable(EnvironmentVariableNamesForAzureSearch.ENDPOINT_SEARCH),
-    key: assertEnvironmentVariable(EnvironmentVariableNamesForAzureSearch.AZURE_API_KEY_SEARCH),
```

Comment:
I doubt that the test works without authentication, did it?

Reply:
I have updated the tests with the authentication property. There are a few other changes to the test suite that I'll update with the new recordings.